sdk/src/main/assets/hcaptcha-form.html

```diff
@@ -53,8 +53,11 @@
         onError: function onError(errCode) {
             return console.log("error: code ".concat(errCode));
         },
-        onLoaded: function onLoaded() {
-            return console.log('cb: challenge or checkbox is visible');
+        onLoaded: function onOpen() {
+            return console.log('cb: challenge is visible');
```

Comment:
Log is not right. Should be `return console.log('cb: api is loaded');`

Reply:
`BridgeObject.onLoaded()` should be called once in `onHcaptchaLoaded`. Currently it is called on every `execute` which is not right.

```js
function onHcaptchaLoaded() {
        try {
            var renderConfig = getRenderConfig();
            hCaptchaID = hcaptcha.render('hcaptcha-container', renderConfig);
            BridgeObject.onLoaded();
            execute(bridgeConfig, renderConfig);
        } catch (e) {
            console.error(e);
            BridgeObject.onError(29);
        }
    }
```

Reply:
Done.

But again onLoaded call on the java side, do loader hiding and do some UI updates. I still think that the right place for this is `open-callback`